content/en/apps/tutorials/application-settings.md

```diff
@@ -0,0 +1,192 @@
+---
+title: "CHT Application Settings"
+linkTitle: Application Settings
+weight: 1
+description: >
+  Managing CHT application settings
+relatedContent: >
+  apps/reference/app-settings
+  apps/guides/performance/replication/#depth
+  core/overview/transitions 
+---
+
+## Purpose of the Tutorial
+
+This tutorial will take you through how to manage the CHT application settings, including;
+
+- Setting user roles and permissions
+- Enabling and disabling transitions
+- Configuring contact hierarchy and configuring replication.
+
+App settings allow you to both persist information that is critical to the application outside the code, and to create profiles that store the preferences for project deployments.
+
+## Brief Overview of Key Concepts
+
+The settings which control CHT apps are defined in the *app_settings.json* file, and stored in the settings doc in the database.
+
+*Permissions* are settings that control access to specific app features and functionality.
+
+*Roles* define permissions for users to perform access a group of app features and functionality.
+
+*Replication* is when users download a copy of the data on to their device. *[Replication depth](https://docs.communityhealthtoolkit.org/apps/guides/performance/replication/#depth)* refers to the number of levels within a hierarchy a specific user role is able to replicate.
+
+*[A transition](https://docs.communityhealthtoolkit.org/core/overview/transitions/)* is javascript code that runs when a document is changed. A transition can edit the changed doc or do anything server side code can do for that matter.
```

Comment:
update transition link to follow style guide: {{< relref "core/overview/transitions" >}}

Delete "A" from beginning of sentence

Reply:
Updated.